external/netstandard20/netstandard20.depproj

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
+      netstandard2.0;
```

Comment:
Naw, I think we'll make this thing support all versions, then just say when restoring for netcoreapp make sure to use net461, or potentially some centrally defined property that says what version of the desktop assemblies that netcoreapp is trying to be compatible with.